### PR TITLE
fix `openid` scope issue as client is requesting for `id_token`

### PR DESCRIPTION
**Pull request by KhizerRehan**
This PR fix the `infinite` loop issue due to missing of providing scope of `openid` As client request is asking for the `response_type=id_token`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,34 +54,48 @@ The build is minified and the filenames include the hashes.<br>
...
 
 4. This client (accessible from `Token Service -> Clients -> client-assisted-example -> Edit Client`) should have the `Implicit Flow` and `Assisted Token` capabilities selected under the `Capabilities` section.
 
+
    ![image](./docs/images/client-capabilities.png)
 
-5. Update the `Redirect URIs` and `Allowed Origins` settings for the `client-assisted-example` Client. The redirect URI should be `http://localhost:3000`. The allowed origin should be the same or, for testing purposes, you can also use `*`.
+
+5. Navigate to `OAuth/OpenID Settings` section make sure to add
+   `opendid` scope.
```
Comment (travisspencer) [inline README.md]: typo

**Review by travisspencer — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (KhizerRehan) [reply README.md]: Good Catch! Thanks

File: README.md [hunk 2]
```diff
@@ -54,34 +54,48 @@ The build is minified and the filenames include the hashes.<br>
...
 
 4. This client (accessible from `Token Service -> Clients -> client-assisted-example -> Edit Client`) should have the `Implicit Flow` and `Assisted Token` capabilities selected under the `Capabilities` section.
 
+
    ![image](./docs/images/client-capabilities.png)
 
-5. Update the `Redirect URIs` and `Allowed Origins` settings for the `client-assisted-example` Client. The redirect URI should be `http://localhost:3000`. The allowed origin should be the same or, for testing purposes, you can also use `*`.
+
+5. Navigate to `OAuth/OpenID Settings` section make sure to add
+   `openid` scope.
```
Comment (Waseem826) [inline README.md]: Move this to previous line

File: README.md [hunk 3]
```diff
@@ -54,34 +54,48 @@ The build is minified and the filenames include the hashes.<br>
...
+   ![image](./docs/images/openid-scope.png)
+
+
+6. Update the `Redirect URIs` and `Allowed Origins` settings for the `client-assisted-example` Client. The redirect URI should be `http://localhost:3000`. The allowed origin should be the same or, for testing purposes, you can also use `*`.
 
    ![image](./docs/images/client-application-settings.png)
+
        
-6. Commit the changes and you are all setup.
+7. `Validate` and `Commit` the changes and you are all setup.
```
Comment (Waseem826) [inline README.md]: No need to say `Validate` here. 
```suggestion
7. Commit the changes and you are all setup.
```

**Review by Waseem826 — CHANGES_REQUESTED** [review]
Screenshot for `openid` scope should be consistent with other screenshots (.i.e no top nav bar and full width)

**Review by Waseem826 — APPROVED** [review]

**Review by travisspencer — APPROVED** [review]

**Pull request merged**